### PR TITLE
Wallet update dependencies

### PR DESCRIPTION
**Pull request by jpcapurro-atixlabs**
Side effect of trying to test the `wasmWrapper` and failing to do so. See: https://github.com/input-output-hk/js-chain-libs/pull/112 for details.

### CODE REVIEW

File: examples/wallet/app/components/SendTransaction.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ export default ({ balance, nodeSettings, sendTransaction }: Props) => {
   const [amount, setAmount] = useState<?number>();
```
Comment (dmernies-iohk) [inline examples/wallet/app/components/SendTransaction.js]: There are no errors in the source code of this component, but when the wallet is executed the send button could have margin right

Comment (dmernies-iohk) [reply examples/wallet/app/components/SendTransaction.js]: feedback

File: examples/wallet/app/containers/App.js [hunk 2]
```diff
@@ -10,6 +10,6 @@ export default class App extends React.Component<Props> {
 
   render() {
     const { children } = this.props;
-    return <React.Fragment>{children}</React.Fragment>;
+    return <>{children}</>;
```
Comment (dmernies-iohk) [inline examples/wallet/app/containers/App.js]: Why do you return an empty tag?

File: examples/wallet/package.json [hunk 3]
```diff
@@ -164,135 +164,135 @@
...
+    "webpack": "4.41.5",
     "webpack-bundle-analyzer": "3.6.0",
-    "webpack-cli": "3.3.0",
-    "webpack-dev-server": "3.2.1",
-    "webpack-merge": "4.2.1",
-    "yarn": "1.19.1"
+    "webpack-cli": "3.3.10",
+    "webpack-dev-server": "3.10.1",
+    "webpack-merge": "4.2.2",
+    "yarn": "1.21.1"
```
Comment (dmernies-iohk) [inline examples/wallet/package.json]: I had to use node 12 to install the dependencies

Comment (jpcapurro-atixlabs) [reply examples/wallet/package.json]: whaaaaat? doesn't it work with node 10?

File: examples/wallet/app/containers/App.js (hunk 2 above)
Comment (jpcapurro-atixlabs) [reply examples/wallet/app/containers/App.js]: It's the same as before, but with a [shorthand syntax](https://reactjs.org/docs/fragments.html#short-syntax) which the new version of eslint prefers.

File: examples/wallet/app/components/SendTransaction.js (hunk 1 above)
Comment (jpcapurro-atixlabs) [reply examples/wallet/app/components/SendTransaction.js]: enzo pointed this out already, I'll address it [after testing the form](https://github.com/input-output-hk/js-chain-libs/pull/116/)

**Pull request merged**